### ISSUE TITLE
Black test fails

### ISSUE THREAD

**Issue by chadell** [nautobot/nautobot-plugin-device-onboarding#34]
### Environment
* Python version:  
* Nautobot version:  
* nautobot-device-onboarding version:  develop branch

### Steps to Reproduce
1. Trigger CI
2.
3.

### Expected Behavior

Black test should pass

### Observed Behavior

```
Run poetry run invoke black
Traceback (most recent call last):
  File "/home/runner/.cache/pypoetry/virtualenvs/nautobot-device-onboarding-pQtf[8](https://github.com/nautobot/nautobot-plugin-device-onboarding/runs/7108360205?check_suite_focus=true#step:4:9)xuE-py3.[9](https://github.com/nautobot/nautobot-plugin-device-onboarding/runs/7108360205?check_suite_focus=true#step:4:10)/bin/black", line 5, in <module>
    from black import patched_main
  File "/home/runner/.cache/pypoetry/virtualenvs/nautobot-device-onboarding-pQtf8xuE-py3.9/lib/python3.9/site-packages/black/__init__.py", line 52, in <module>
    from typed_ast import ast3, ast27
  File "/home/runner/.cache/pypoetry/virtualenvs/nautobot-device-onboarding-pQtf8xuE-py3.9/lib/python3.9/site-packages/typed_ast/ast3.py", line 40, in <module>
    from typed_ast import _ast3
ImportError: /home/runner/.cache/pypoetry/virtualenvs/nautobot-device-onboarding-pQtf8xuE-py3.9/lib/python3.9/site-packages/typed_ast/_ast3.cpython-39-x86_64-linux-gnu.so: undefined symbol: _PyUnicode_DecodeUnicodeEscape
```

**Comment by glennmatthews** [comment]
Simplest fix will likely be upgrading the installed Black version.

**Comment by chadell** [comment]
> Simplest fix will likely be upgrading the installed Black version.

yes, `poetry lock` is still `resolving dependencies...` 😞

**Comment by chadell** [comment]
I believe fixed in #35

**Comment by scetron** [comment]
Fixed. Black tests no longer failing in CI or locally.
